schedule/functional/zdup_Leap_15.x_kde.yaml

```diff
@@ -0,0 +1,77 @@
+---
+name: zdup from leap15.5+ kde to tumbleweed or new leap version
+description: >
+    Maintainer: qe-core
+    patch leap system before upgrade
+schedule:
+    - installation/bootloader_start
```

Comment:
IMO this should not use the yaml schedule, main.pm is more suitable for TW.

Reply:
Thanks, then let me use main.pm

Reply:
(Feel free to wait for other's opinion on this though)

Reply:
I'm here with @Vogtinator